### PR TITLE
added solution

### PR DESCRIPTION
**Pull request by matrix880**

### CODE REVIEW

File: src/main/java/core/basesyntax/FieldXY.java [hunk 1]
```diff
@@ -14,6 +14,34 @@
 
     public void moveRobot(Robot robot, int toX, int toY) {
         //your code
+
+        if (robot.getX() > toX) {
+
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/FieldXY.java:19]: remove empty line

File: src/main/java/core/basesyntax/FieldXY.java [hunk 2]
```diff
@@ -14,6 +14,34 @@
...
+            while (robot.getDirection() != Direction.LEFT) {
+                robot.turnLeft();
+            }
+        } else {
+
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/FieldXY.java:24]: remove empty line

File: src/main/java/core/basesyntax/FieldXY.java (hunk 1 above)
Comment (matrix880) [reply src/main/java/core/basesyntax/FieldXY.java]: done

File: src/main/java/core/basesyntax/FieldXY.java (hunk 2 above)
Comment (matrix880) [reply src/main/java/core/basesyntax/FieldXY.java]: done

**Pull request closed without merging**